src/qibolab/serialize.py

```diff
@@ -205,6 +205,35 @@ def dump_runcard(platform: Platform, path: Path):
         platform.qubits, platform.couplers
     )
 
-    path.write_text(
+    (path / RUNCARD).write_text(
         yaml.dump(settings, sort_keys=False, indent=4, default_flow_style=None)
     )
+
+
+def dump_kernels(platform: Platform, path: Path):
+    """Creates Kernels instance from platform and dumps as npz.
+
+    Args:
+        platform (qibolab.platform.Platform): The platform to be serialized.
+        path (pathlib.Path): Path that the kernels file will be saved.
+    """
+
+    # create and dump kernels
+    kernels = Kernels()
+    for qubit in platform.qubits.values():
+        if qubit.kernel is not None:
+            kernels[qubit.name] = qubit.kernel
+
+    kernels.dump(path / KERNELS)
+
+
+def dump_platform(platform: Platform, path: Path):
+    """Platform serialization as runcard (yaml) and kernels (npz).
+
+    Args:
+        platform (qibolab.platform.Platform): The platform to be serialized.
+        path (pathlib.Path): Path where yaml and npz will be dumped.
+    """
+
```

Comment:
Didn’t try to use this yet, but shouldn’t it:
1. Create the folder specified by `path` if it doesn’t exist (given that it’ll try to write files inside)
2. Dump also `platform.py` inside this folder?

I’ll try to use and review this comment.